README.md

```diff
@@ -52,13 +52,24 @@ require one for your use case.
 * [virtualenv](https://pypi.org/project/virtualenv/)
 * [venv](https://docs.python.org/3/library/venv.html)
 
+NOTE: MacOS sometimes has issues with installing since it comes with python2.7 by default.
+
+### via `homebrew`
+
+[brew](https://brew.sh/)
+If you are installing ape on a MacOS. Most users find it easier to install it on brew. It is the same process as pip.
+
+* (MacOS Alternative Option) Install via homebrew [brew](https://formulae.brew.sh/formula/virtualenv)
+
 Create your virtual environment folder
 
 ```bash
 python3 -m venv /path/to/new/environment
 source <venv_folder>/bin/activate
 ```
 
+NOTE: the path to venv is not a real path.
```

Comment:
I think that is obvious, probably don't need to mention

Reply:
WELL I THOUGHT IT WAS. BUT IVE BEEN SENT SCREENSHOTS WITH PEOPLE NOT GETTING IT
😆